src/Css/Style.php

```diff
@@ -1818,6 +1650,11 @@ protected function _set_style_side_type($style, $side, $type, $val, $important)
         $this->_props[$prop] = $val;
```

Comment:
Some properties are dependent on the font size. As a result, when the font size changes those properties are re-calculated (using the generic setter to avoid inconsistent behavior). You can see the dependencies in the `$_dependency_map` static property.

We only re-calculate dependent properties that have already been set, and only within the current style instance. It should be a minor performance impact, though every bit of recursion can be problematic with a large enough sample size.

I was toying with a change to the style declaration parsing logic where we only set the specified property during parsing and, when that was complete, calculate the computed values. I dropped it due to some complexity in implementation, though we could pick it up again later.

Reply:
@bsweeney 
Do you mean we have to live with crashes in this scenario at the moment? I don't know how likely it is to have those cases. In my tests it just happens if I set the value inhertit but maybe it can happen in other cases too.
Initially it appeared when loading the html from page "https:/encode.su"

Reply:
oh, no ... that's not what I meant at all. Let me take a look at this again.

Reply:
If I had actually run your sample instead of just read and interpreted I would have seen the issue. Thanks for catching that. I addressed the issue with #2142.